Chatify/chat/consumers.py

```diff
@@ -57,10 +54,31 @@ async def connect(self):
     async def disconnect(self, close_code):
         await self.channel_layer.group_discard(self.room_group_name, self.channel_name)
 
+    @sync_to_async
+    def send_data_to_save_chat(self, sender, msg, receiver):
+        from .models import Group, Chat, User
+
+        sender_id = User.objects.get(id=int(sender))
+        receiver_id = User.objects.get(id=int(receiver))
+        try:
+            instance = Group.objects.get(name=self.group_name)
+        except Group.DoesNotExist:
+            instance = Group.objects.create(name=self.group_name)
+        instance.user.set(sender)
```

Comment:
why are you writing this line ?

Reply:
Because in Group Model user field has ManytoMany field that's why i have to write this way to set a user